Parsers/ASimProcessEvent/Parsers/imProcessEvent.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
please update this file as well:
https://github.com/Azure/Azure-Sentinel/blob/23b502e285b220f319d6ebea53118a10e8025c79/Parsers/ASimProcessEvent/Parsers/imProcessCreate.yaml

Reply:
DONE

Reply:
added